.eslintrc

```diff
@@ -0,0 +1,60 @@
+{
+    "parser": "babel-eslint",
+    "plugins": ["react", "react-hooks", "babel"],
+    "parserOptions": {
+      "ecmaVersion": 6,
+      "sourceType": "module",
+      "ecmaFeatures": {
+        "jsx": true
+      }
+    },
+    "env": {
+      "browser": true,
+      "amd": true,
+      "es6": true,
+      "node": true,
+      "mocha": true
+    },
+    "extends": "airbnb",
+    "rules": {
+      "camelcase": 0,
+      "arrow-parens": 0,
+      "arrow-body-style": 0,
+      "class-methods-use-this": 0,
+      "import/imports-first": 0,
+      "import/no-extraneous-dependencies": 0,
+      "import/prefer-default-export": 0,
+      "import/newline-after-import": 0,
+      "import/no-unresolved": 0,
+      "import/no-named-as-default": 0,
+      "jsx-a11y/img-has-alt": 0,
+      "jsx-a11y/no-static-element-interactions": 0,
+      "no-unused-vars": 1,
+      "global-require": 0,
+      "comma-dangle": 0,
```

Comment:
I was also same as you about turning this rule off when I used airbnb eslint rules in the first beginning.

But, after playing with it for a while, I found it do have some benefits.

For example, while you have an object with several attributes, when you'd like to add more attributes to it, you'll only have diff for the new attributes.

Reply:
Thanks a lot for the advice of linting rules! I think I'll see through these rules once again, and decide which I'll skip